nodeletctl/pkg/nodeletctl/nodeletctl.go

```diff
@@ -15,72 +15,72 @@ import (
 )
 
 type BootstrapConfig struct {
-	SSHUser                string                 `json:"sshUser,omitempty" survey:"sshUser"`
-	SSHPrivateKeyFile      string                 `json:"sshPrivateKeyFile,omitempty" survey:"sshPrivateKeyFile"`
-	CertsDir               string                 `json:"certsDir,omitempty" survey:"certsDir"`
-	KubeConfig             string                 `json:"kubeconfig,omitempty" survey:"kubeconfig"`
-	Pf9KubePkg             string                 `json:"pf9KubePkg,omitempty" survey:"pf9KubePkg"`
-	ClusterId              string                 `json:"clusterId,omitempty" survey:"clusterId"`
-	AllowWorkloadsOnMaster bool                   `json:"allowWorkloadsOnMaster,omitempty" survey:"allowWorkloadsOnMaster"`
-	K8sApiPort             string                 `json:"k8sApiPort,omitempty" survey:"k8sApiPort"`
-	MasterIp               string                 `json:"masterIp,omitempty" survey:"masterIp"`
-	MasterIpv6             string                 `json:"masterIpV6,omitempty" survey:"masterIpV6"`
-	MasterVipEnabled       bool                   `json:"masterVipEnabled,omitempty" survey:"masterVipEnabled"`
-	MasterVipInterface     string                 `json:"masterVipInterface,omitempty" survey:"masterVipInterface"`
-	MasterVipVrouterId     int                    `json:"masterVipVrouterId,omitempty" survey:"masterVipVrouterId"`
-	MTU                    string                 `json:"mtu,omitempty" survey:"mtu"`
-	Privileged             string                 `json:"privileged,omitempty" survey:"privileged"`
+	SSHUser                string                 `json:"sshUser,omitempty" yaml:"sshUser,omitempty" survey:"sshUser"`
+	SSHPrivateKeyFile      string                 `json:"sshPrivateKeyFile,omitempty" yaml:"sshPrivateKeyFile,omitempty" survey:"sshPrivateKeyFile"`
+	CertsDir               string                 `json:"certsDir,omitempty" yaml:"certsDir,omitempty" survey:"certsDir"`
+	KubeConfig             string                 `json:"kubeconfig,omitempty" yaml:"kubeconfig,omitempty" survey:"kubeconfig"`
+	Pf9KubePkg             string                 `json:"nodeletPkg,omitempty" yaml:"nodeletPkg,omitempty" survey:"nodeletPkg"`
```

Comment:
So the most important change is to rename `pf9KubePkg` to `nodeletPkg`

Reply:
No. The yaml part is important change.

Reply:
Do we need to backport the YAML part to LTS1 too?

Reply:
No I think, not needed. This is only for smcp.